node/src/Pos/Wallet/Web/ClientTypes/Types.hs

```diff
@@ -295,7 +295,7 @@ data CTExMeta = CTExMeta
 data CUpdateInfo = CUpdateInfo
     { cuiSoftwareVersion :: !SoftwareVersion
     , cuiBlockVesion     :: !BlockVersion
-    , cuiScriptVersion   :: !ScriptVersion
+    , cuiScriptVersion   :: !(Maybe ScriptVersion)
```

Comment:
Doesn't it break frontend?

Reply:
It probably does, I don't know how to run front-end to test.

Reply:
Indeed
We should pass `ScriptVersion` here, I guess. Take it from currently adopted block version `X.Y.Z`. This is safe because block version modifier modifies only currently adopted version (which is unique) and once adopted version changes, old proposals based on previous adopted version would become invalid (and this is enforced by versioning scheme)

Reply:
Or just change frontend code accordingly (which is probably easier). You can ask akegalj about details I suppose.